financialaid/models.py

```diff
@@ -0,0 +1,110 @@
+"""
+Models for the Financial Aid App
+"""
+
+from django.contrib.auth.models import User
+from django.db import (
+    models,
+    transaction,
+)
+from jsonfield import JSONField
+
+from courses.models import (
+    CourseRun,
+    Program,
+)
+
+
+class Tiers(models.Model):
```

Comment:
these model classes should be singular, not plural (~~Tiers~~ -> Tier, ~~Coupons~~ -> Coupon, ~~Documents~~ -> Document)